src/styles/styles.css

```diff
@@ -0,0 +1,315 @@
+@charset "UTF-8";
+
+/* VARIABLES HEADER */
+
+/* VARIABLES CARD */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+/* #region HEADER */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__nav {
+  padding: 0;
+}
+
+.header__nav-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.header__nav-item {
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header__nav-item:last-child {
+  margin-right: 0;
+}
+
+.header__nav-item__link {
+  color: #060b35;
+  text-decoration: none;
+  display: block;
+  line-height: 60px;
+}
+
+.header__nav-item__link:hover,
+.header__nav-item__link:active {
+  color: #00acdc;
+}
+
+.header__nav-item__link:active {
+  text-decoration: underline;
```

Comment:
The `text-decoration` property should be defined in a variable. This helps to maintain the code and makes it easier to change the decoration in the future if needed.